src/libs/ReportActionsUtils.ts

```diff
@@ -191,18 +191,22 @@ function getWhisperedTo(reportAction: OnyxInputOrEntry<ReportAction>): number[]
     const originalMessage = getOriginalMessage(reportAction);
     const message = getReportActionMessage(reportAction);
 
-    if (!(originalMessage && 'whisperedTo' in originalMessage) && !(message && 'whisperedTo' in message)) {
+    if (!(originalMessage && typeof originalMessage === 'object' && 'whisperedTo' in originalMessage) && !(message && typeof message === 'object' && 'whisperedTo' in message)) {
         return [];
     }
 
     if (!Array.isArray(message) && typeof message === 'object' && 'whisperedTo' in message) {
         return message?.whisperedTo ?? [];
     }
 
-    if (originalMessage && 'whisperedTo' in originalMessage) {
+    if (originalMessage && typeof originalMessage === 'object' && 'whisperedTo' in originalMessage) {
         return originalMessage?.whisperedTo ?? [];
     }
 
+    if (typeof originalMessage !== 'object') {
+        Log.info('Original message is not an object for reportAction: ', true, reportAction);
```

Comment:
We cannot log the entire report action, that has sensitive data. only its ID and type

Reply:
There are some places in `ReportActionUtils` where the `reportAction` is logged like [here](https://github.com/Expensify/App/blob/060160e892b9bfaddab75614817b4b906a2c7b3f/src/libs/ReportActionsUtils.ts#L581-L592) so I followed that.

I will push a commit.

Reply:
```js
if (typeof originalMessage !== 'object') {
        Log.info('Original message is not an object for reportAction: ', true, {"reportActionID" : reportAction?.reportActionID, "actionName" : reportAction?.actionName});
    }
```

This is fine, right?

Reply:
We probably redact the sensitive fields in the backend but still safer to just send the reportActionID